src/components/playlist/video/videoDescription.tsx

```
@@ -0,0 +1,19 @@
+import Card from '@/components/common/card';
+
+type VideoDescriptionProps = {
+    description?: string | undefined;
+}
+
+const VideoDescription = ({
+    description
+}: VideoDescriptionProps) => {
+    return (
+        <Card>
```

ESLint

[error] 'React' must be in scope when using JSX

(react/react-in-jsx-scope)